### PR TITLE
Set up basic infrastructure

### PR DESCRIPTION
**Pull request by lpil**
# Motivation

We want a website.

Specific tasks- 
https://trello.com/c/wyyugBs8

## Checklist

- [x] Basic project structure
- [x] Cloudformation stack
- [x] Lambda function creation
- [x] S3 bucket creation
- [x] Cloudfront distribution creation

### CODE REVIEW

**Comment by zoeabryant** [comment]
todo:
- [ ] remove references to your name

- [ ] basic **publish service**

- [ ] basic **dev server**

- [ ] basic **site component**

File: assets/badger.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                       .mmmmmmmmmmmmmmmmmmmmmmmmmmmmmm/
+                        ommmmmmmmmmmmmmmmmmmmmmmmmmmm/
+                        `ymmmmmmmmmmmmmmmmmmmmmmmmmy.
+                         `ymmmmmmmmmmmmmmmmmmmmmmmy
+                           +mmmmmmmmm/::mmmmmmmmms`
+                            .smmmmmmm- .mmmmmmmd:
+                              `/osyys. `oyyyso-
+
+                              Red Badger Website.
+                                   Honestly.
```
Comment (zoeabryant) [inline assets/badger.txt]: 👍

Comment (asavin) [inline assets/badger.txt]: I think we should keep this ASCII art, but change ASCII art on BB and React.London to something different.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
-Serverless Static Site
-======================
+Website. Honestly.
+==================
 
-Static site compilation on AWS Lambda.
+Red Badger Website Episode 6: Return of the Jedi.
 
+
+## Usage
```
Comment (asavin) [inline README.md]: Should probably setup `/docs` folder straight away

File: bin/publish-service.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SERVICE_DIR=$(realpath ./publish-service)
+NODE_BIN=$(realpath ./node_modules/.bin)
+SERVERLESS=$NODE_BIN/serverless
+
+ENV_FILE=.env
+VAR_NAMES=$(cat $ENV_FILE | grep = | cut -d= -f1)
```
Comment (AndrewBestbier) [inline bin/publish-service.sh]: WRONG!!!!! ;)

**Pull request merged**